### PR TITLE
[14.0] [ADD] purchase_order_partner_manual_rank

### PR DESCRIPTION
**Pull request by renda-dev**
Simple module that adds a filter on purchase order vendor to show only partners that are suppliers.

### CODE REVIEW

File: purchase_order_partner_manual_rank/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+This module adds a filter on purchase order vendor
+to show only partners that are suppliers.
```
Comment (francesco-ooops) [inline purchase_order_partner_manual_rank/readme/DESCRIPTION.rst]:
```suggestion
to show only partners that are suppliers as per module partner_manual_rank behavior.
```

**Review by GSLabIt — APPROVED** [review]
code review ok

**Review by francesco-ooops — APPROVED** [review]
Functional ok!

**Comment by francesco-ooops** [comment]
@OCA/procurement-purchase-maintainers merge? :)

**Comment by francesco-ooops** [comment]
@yajo could you please take a look? :)

**Review by yajo — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**